### PR TITLE
move_to_end function was improved

### PR DESCRIPTION
**Pull request by AnnaCalderon**
move_to_end function was improved to detect if "labelled" is before "hms" even if labelled is not the first element. Addressing the warning in line 100.

### CODE REVIEW

**Review by RaymondBalise — CHANGES_REQUESTED** [review]
Plase lint and style the code

**Comment by RaymondBalise** [comment]
Please add our orcID to the description file

**Comment by AnnaCalderon** [comment]
>

**Comment by AnnaCalderon** [comment]
lint and style the code donde

**Comment by AnnaCalderon** [comment]
orcID added in a separate branch and merged.

File: R/make_instrument_auto.R [hunk 1]
```diff
@@ -89,15 +89,24 @@ fix_class_bug <- function(df) {
   classes_ls <- lapply(df, class)
   
   # Creating generic function to move first element of vector to the end
+  # if the first element is "labelled"
   move_to_end <- function(x) {
-    c(x[2:length(x)], x[1])
+    if("labelled" %in% x & "hms" %in% x) {
+      if(which(x =="hms") > which(x == "labelled")){
```
Comment (RaymondBalise) [inline R/make_instrument_auto.R:95]: Please lint the code.  Notice the missing space between =="hms"

**Review by RaymondBalise — CHANGES_REQUESTED** [review]

**Pull request merged**